### PR TITLE
🔄 sync collections search

### LINKED ISSUE #1687: Search is missing on the collection page

### PR DESCRIPTION
**Pull request by roiLeo**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [x] Bugfix

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### What's new?
- [x] added collections searchbar
- [x] PR closes #1687 

### Screenshot
Screenshot 2022-01-04 at 10-21-27 KodaDot - Polkadot Kusama NFT explorer

### CODE REVIEW

File: pages/rmrk/collections.vue [hunk 1]
```diff
@@ -134,8 +154,11 @@ export default class Collections extends mixins(PrefixMixin) {
       result: this.handleResult,
       variables: () => {
         return {
+          orderBy: this.searchQuery.sortBy,
+          search: this.buildSearchParam(),
+          listed: this.searchQuery.listed ? [{price: { greaterThan: '0'}}] : [],
           first: this.first,
-          offset: this.offset,
+          offset: this.offset
```
Comment (vikiival) [inline pages/rmrk/collections.vue:161]:
```suggestion
          offset: this.offset,
```

**Comment by vikiival** [comment]
Legit! Works!
Screenshot 2022-01-04 at 10 34 35

However @roiLeo can we centre the buy button?

**Review by vikiival — APPROVED** [review]

**Comment by roiLeo** [comment]
> Legit! Works! Screenshot 2022-01-04 at 10 34 35
> 
> However @roiLeo can we centre the buy button?

#1685 I think it will be done here?

**Comment by vikiival** [comment]
Or let it merge 🥇

**Pull request merged**